lib/queryCompiler.ts

```diff
@@ -162,6 +162,19 @@ export abstract class QueryCompiler extends QueryVisitor<CompilationContext> {
         return context;
     }
 
+    protected visitInPredicate: CompileMethod<ast.InPredicate> = (context, node) => {
+        context.queryString =
+            "("
+            + this.visitNode(context, node.predicate).queryString
+            + (node.negate ? " NOT IN " : " IN ")
+            + "("
+            + this.visitNode(context, node.target).queryString
```

Comment:
Does this cover the IN predicate with an array as well? Like SELECT * FROM a WHERE a.category IN ('Cat1', 'Cat2', 'Cat3')?